nvidia-bert/ort_addon/ort_supplement/ort_supplement.py

```diff
@@ -43,99 +50,94 @@ def setup_onnxruntime_with_mpi(args):
 
         torch.distributed.init_process_group(backend='nccl')
 
-    from onnxruntime.capi._pybind_state import set_cuda_device_id 
+    # tell onnxruntime which device to use and seed its random generators
+    from onnxruntime.capi._pybind_state import set_cuda_device_id, set_seed
```

Comment:
`onnxruntime.capi._pybind_state` is an internal namespace and we shouldn't use it

use `from onnxruntime import set_seed` instead

You probably don't need to use `set_cuda_device_id` as you already set device ID on line 104 (ORTTRainerOptions.device.id)

Reply:
Hmm, I got the following..
```
File "/workspace/bert/ort_supplement/ort_supplement.py", line 58, in setup_onnxruntime_with_mpi
    from onnxruntime import set_arena_extend_strategy, ArenaExtendStrategy
ImportError: cannot import name 'set_arena_extend_strategy' from 'onnxruntime' (/opt/conda/lib/python3.7/site-packages/onnxruntime/__init__.py)
```

It's not expected to be exposed? I'm not sure how much space it saves. I can probably just remove it.

Reply:
set_arena_extend_strategy is not exposed yet. if you need it, we can expose it

Reply:
I see, OK. I think it's fine to remove it. I'm not sure the C++ BERT run sets that either. In any case, it's better suited for next PR targeting performance.